README.md

```diff
@@ -40,34 +40,31 @@ If the SFTP service is used, the question arises regarding how the system will k
 
 1. Clone this repository.
 2. Change directories into the `base-local` directory.
-3. Determine whether to store encrypted files on the local file system or in AWS S3 (*hint: for this initial version, choose filesystem*).
-4. Copy the proper files based on the storage selection; for example:
+3. Adjust if needed values in [.env file](./env)
+4. Determine whether to store encrypted files on the local file system or in AWS S3 (*hint: for this initial version, choose filesystem*).
+5. Copy the proper files based on the storage selection; for example:
     ```bash
     cp docker-compose.yaml.fs docker-compose.yaml
-    cp files/etc/config.yaml.fs files/etc/config.yaml
+    cp etc/config.yaml.fs etc/config.yaml
     ```
-5. Edit the IP address hard-coded in the docker-compose.yaml file to match that of your local host. Also review the port numbers and the passwords provided and change if needed.
 6. When ready, run the Docker Compose command in the base-local directory to instantiate the Docker stack:
     ```bash
     sudo docker-compose up -d
     ```
-7. When the containers are built and deployed, the database is empty; import the database schemas:
+7. Wait till the database starts and is ready to connect to using the following command:
     ```bash
-    sudo ./populate_mysql_databases.sh
+    docker-compose logs -f database
```

Comment:
If installed systemwide, the docker-compose command would need a sudo prefix (mine is). Also, the -f is probably not needed here. When I ran it, I didn't catch the `-f` bit initially and was waiting for the command to complete (when it's actually following the output). Getting the initial output shows the sql files are loaded successfully.

Reply:
I updated it accordingly